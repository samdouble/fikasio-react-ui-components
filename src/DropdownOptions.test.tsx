import React from 'react';
import renderer from 'react-test-renderer';
import { DropdownOptions } from '.';

describe('DropdownOptions', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<DropdownOptions />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
