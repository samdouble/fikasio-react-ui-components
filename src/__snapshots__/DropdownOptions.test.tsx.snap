// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownOptions Renders correctly 1`] = `
<div
  className="fikasio-dropdownOptions fikasio-theme-dark"
  style={Object {}}
>
  <div
    className="dropdown"
    style={
      Object {
        "position": "static",
      }
    }
  >
    <div
      className="fikasio-dropdownOptions-toggle fikasio-theme-dark dropdown-toggle"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      style={Object {}}
      tabIndex={0}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-ellipsis fa-1x "
        data-icon="ellipsis"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 256a56 56 0 1 1 112 0A56 56 0 1 1 8 256zm160 0a56 56 0 1 1 112 0 56 56 0 1 1 -112 0zm216-56a56 56 0 1 1 0 112 56 56 0 1 1 0-112z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
  </div>
</div>
`;
